src/app/change-account-selector/change-account-selector.component.html

```diff
@@ -1,17 +1,19 @@
 <div
-  class="fs-15px fc-default px-5px cursor-pointer change-account-selector br-12px"
+  class="fs-14px fc-default px-5px cursor-pointer change-account-selector br-4px change-account-selector__hover"
   [ngClass]="{ 'change-account-selector__shadow': selectorOpen }"
   #changeAccountSelectorRoot
   id="changeAccountSelectorRoot"
 >
   <!-- Selector (unopened state) -->
-  <div class="d-flex align-items-center justify-content-between pl-10px pr-10px change-account-selector__hover">
-    <div *ngIf="globalVars.loggedInUser" class="change-account-selector__ellipsis-restriction cursor-pointer">
+  <div class="d-flex align-items-center justify-content-between pl-5px pr-5px">
+    <div *ngIf="globalVars.loggedInUser" class="change-account-selector__ellipsis-restriction d-flex flex-row align-items-center cursor-pointer">
+      <div class="feed-create-post__avatar align-items-center change-account-selector__account-image" [avatar]="globalVars.loggedInUser.PublicKeyBase58Check"></div>
+      <div class="change-account-selector__acount-name">
```

Comment:
```suggestion
      <div class="change-account-selector__acount-name change-account-selector__ellipsis-restriction">
```

when a long name is selected, it currently doesn't show ellipsis. this should resolve that.